elastic/kibana
# [Alerting UI] Not showing edit button in rule management UI if rule is not editable in UI

Resolves https://github.com/elastic/kibana/issues/107324

## Summary

Edit button was showing for rules that were not editable in the stack management UI. Applied the same logic as used on the rule details page to the rule list view.

## To Verify

Create a security detection rule. Navigate to Rules & Connectors UI and verify that the edit button does not show up on hover on the rule list and that the Edit action is disabled in the menu dropdown. Verify that other rule types are editable from the rule list page.

### Checklist

Delete any items that are not applicable to this PR.
